syntax = "proto3";

package app;

// Define request and response messages
message DummyRequest {
    string message = 1;  // Add a field for demonstration
}

message DummyResponse {
    string response = 1;  // Add a field for demonstration
}

// Define the service with input and output types
service DashService {
    rpc RunDashApp(DummyRequest) returns (DummyResponse);
}
